src/sentry/auth/superuser.py

```diff
@@ -93,7 +94,7 @@ class EmptySuperuserAccessForm(SentryAPIException):
 
 
 class Superuser:
-    allowed_ips = [ipaddress.ip_network(str(v), strict=False) for v in ALLOWED_IPS]
+    allowed_ips = frozenset(ipaddress.ip_network(str(v), strict=False) for v in ALLOWED_IPS)
```

Comment:
we only use `allowed_ips` below like so
https://github.com/getsentry/sentry/blob/694b980ceea27b14721c799dae408658c6277f39/src/sentry/auth/superuser.py#L162

Reply:
oh it was before I fixed it